README.md

```diff
@@ -63,12 +66,10 @@ the RFC process, it may be closed with a polite request to submit an RFC first.
 [Sub-team specific guidelines]: #sub-team-specific-guidelines
 
 For more details on when an RFC is required for the following areas, please see
-the Rust community's [sub-team] specific guidelines for:
+the Sawtooth community's [sub-team] specific guidelines for:
 
-  - [language changes](lang_changes.md),
-  - [library changes](libs_changes.md),
-  - [compiler changes](compiler_changes.md).
 
+  - [core changes](core_changes.md)
```

Comment:
Do we need subteams at all if we only have the one? Is this referencing a community infrastructure we aren't actually going to build out?

Reply:
Yes, see other comment (sub-teams will be specified in an RFC).